go.sum

```diff
@@ -415,6 +415,8 @@ github.com/stretchr/testify v1.9.0 h1:HtqpIVDClZ4nwg75+f6Lvsy/wHu+3BoSGCbBAcpTsT
 github.com/stretchr/testify v1.9.0/go.mod h1:r2ic/lqez/lEtzL7wO/rwa5dbSLXVDPFyf8C91i36aY=
 github.com/subosito/gotenv v1.6.0 h1:9NlTDc1FTs4qu0DDq7AEtTPNw6SVm7uBMsUCUjABIf8=
 github.com/subosito/gotenv v1.6.0/go.mod h1:Dk4QP5c2W3ibzajGcXpNraDfq2IrhjMIvMSWPKKo0FU=
+github.com/tjons/api v0.0.0-20241022224639-7a6e49ca3102 h1:p0rKANEJjG+NJl8303bZ28X3SbwFLCUyuAfNNOLFt0s=
```

Comment:
???

Reply:
this is WIP - untangling merge changes, please bear with me and I will fix shortly!